mf-polis/dme/dme-import-map.json

```diff
@@ -0,0 +1,24 @@
+{
+  "imports": {
+    "single-spa": "https://cdn.jsdelivr.net/npm/single-spa@5.9.0/lib/system/single-spa.min.js",
+    "@single-spa/welcome": "https://unpkg.com/single-spa-welcome/dist/single-spa-welcome.js",
+    "vue": "https://cdn.jsdelivr.net/npm/vue@2.6.14/dist/vue.min.js",
+    "vue-router": "https://cdn.jsdelivr.net/npm/vue-router@3.5.3/dist/vue-router.min.js",
+    "vuetify": "https://cdn.jsdelivr.net/npm/vuetify@2.6.5/dist/vuetify.min.js",
+    "@digital-enabler/sidebar": "https://cdn.jsdelivr.net/npm/@digital-enabler/sidebar@1.2.0/app.js",
+    "@digital-enabler/footer": "https://cdn.jsdelivr.net/npm/@digital-enabler/footer@1.0.1/app.js",
+    "@digital-enabler/auth": "https://cdn.jsdelivr.net/npm/@digital-enabler/auth@1.2.1/app.js",
+    "@digital-enabler/demf-dme-event-handler": "https://cdn.jsdelivr.net/npm/@digital-enabler/demf-dme-event-handler@1.0.0/app.js",
+    "@digital-enabler/mashups-list": "https://cdn.jsdelivr.net/npm/@digital-enabler/mashups-list@1.2.3/app.js",
+    "@digital-enabler/mashups-editor": "https://cdn.jsdelivr.net/npm/@digital-enabler/mashups-editor@1.0.4/app.js",
+    "@digital-enabler/mashups-editor-appbar": "https://cdn.jsdelivr.net/npm/@digital-enabler/mashups-editor-appbar@1.0.7/app.js",
+    "@digital-enabler/share-form": "https://cdn.jsdelivr.net/npm/@digital-enabler/share-form@1.0.0/app.js",
+    "@digital-enabler/dme-datasources-list": "https://cdn.jsdelivr.net/npm/@digital-enabler/dme-datasources-list@1.1.0/app.js",
+    "@digital-enabler/dme-datasource-details": "https://cdn.jsdelivr.net/npm/@digital-enabler/dme-datasource-details@1.1.3/app.js",
+    "@digital-enabler/dme-flows-list": "https://cdn.jsdelivr.net/npm/@digital-enabler/dme-flows-list@1.2.3/app.js",
+    "@digital-enabler/dme-flow-editor": "https://cdn.jsdelivr.net/npm/@digital-enabler/dme-flow-editor@1.8.4/app.js",
+    "@digital-enabler/demf-custom-operators": "https://cdn.jsdelivr.net/npm/@digital-enabler/demf-custom-operators@0.1.0/app.js",
+    "@digital-enabler/demf-custom-operator-details": "https://cdn.jsdelivr.net/npm/@digital-enabler/demf-custom-operator-details@0.0.4/app.js",
+    "@digital-enabler/dme-gui": "https://mashupsui.dev.dev-digital-enabler.eng.it/demf-dme-gui.js"
```

Comment:
"https://mashupsui.dev.dev-digital-enabler.eng.it/demf-dme-gui.js" replace 'dev' with 'polis'  ?